client/asset/btc/btc.go

```diff
@@ -55,6 +56,7 @@ var (
 		Name:              "Bitcoin",
 		Units:             "Satoshis",
 		DefaultConfigPath: dexbtc.SystemConfigPath("bitcoin"),
+		ConfigOpts:        config.Options(&dexbtc.Config{}),
```

Comment:
Mixed feelings about trying to parse this info from the struct tags using `reflect`. Seems like it would be simpler to reason and more extensible if you defined the `Options` here.

Reply:
🤔
One of the goals for using reflection is to ensure that the `Options` slice is consistent with what's defined in the asset's `Config` struct. Tests could ensure that as well, so that weakens that goal.

I'm inclined to agree with this thinking. I'll think about it some more to be sure I'm not forgetting anything.

Reply:
I thought about it and these 2 thoughts came to mind:
- While we could write tests to ensure that each asset's `ConfigOpts` is consistent with its `Config` struct, we'd have to write this test for every asset added to dexc. Would mostly just be copy-paste from some other asset's test file but we could avoid it entirely by using `reflect`.
- Using `reflect` helps us have one source of truth when it comes to an asset's configuration parameters - the `Config` struct. If we defined the `ConfigOpts` directly, we'd have to update the definition every time we update the `Config` struct (of course tests would help ensure that we do not forget to do so).

If we consider an asset's `Config` struct as the source of truth for configuration parameters, it would not be necessary to make this `ConfigOpts` assignment simpler to reason. A code reader can/would defer to the struct tags on each asset's `Config` struct to understand the nature of required parameters.

That said, there isn't really much difference between both approaches I think. I can change approaches if desired. What do you think?

Reply:
Single source of truth is certainly the pro. The con is that the `ini` tag can only hold a limited amount of information. For example, if we wanted the file contents instead of the file path for the  a wallet's TLS certificate, I don't know if we can accommodate that with the current scheme.

Reply:
I think it can, but I guess we can easily dump the ini-tag-style of defining config options if it turns out to be unusable.